.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
   check_source:
     name: check_source
     runs-on: ubuntu-20.04
-    container: golangci/golangci-lint:v1.42.0
+    container: golangci/golangci-lint:v1.43.0
```

Comment:
Change this to 1.43 instead of 1.43.0.  Without the ".0" it is supposed to take the latest version in the series.  We had deliberately changed it to 1.42.0 because 1.42.1 was breaking something.  Sylabs is using 1.43, so let's do the same.